app/controllers/extraction_definitions_controller.rb

```diff
@@ -15,10 +15,7 @@ def create
     @extraction_definition = ExtractionDefinition.new(extraction_definition_params)
 
     if @extraction_definition.save
-      @harvest_definition.update(extraction_definition_id: @extraction_definition.id)
-
-      2.times { Request.create(extraction_definition: @extraction_definition) }
-
+      setup_new_extraction_definition
```

Comment:
since this has a flow on effect the convention would be to have a ! in the method name